crates/mdman/src/bin/build-man.rs

```diff
@@ -0,0 +1,110 @@
+use std::fs;
```

Comment:
Is there a reason we put this in `mdman`?

I figured us having specific top-level `xtask-` packages would help raise visibility on what xtasks exist

Reply:
Good point. I have no idea what I was doing.

BTW, do you feel like we should probably have only crates meant to be published stay in `crates/`, and move others elsewhere?